### PR TITLE
header wrap fix

### PR DESCRIPTION
**Pull request by amacwhirter**
Possible solution to issue #611

### CODE REVIEW

File: src/components/TableHeadCell.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ const defaultHeadCellStyles = theme => ({
     cursor: 'pointer',
     paddingLeft: '4px',
     height: '10px',
-  },
+    position: 'absolute'
+},
```
Comment (gabrielliwerant) [inline src/components/TableHeadCell.js]: Needs a tab here to keep the bracket in line with the others.

**Comment by gabrielliwerant** [comment]
Fixes https://github.com/gregnb/mui-datatables/issues/611.

File: src/components/TableHeadCell.js (hunk 1 above)
Comment (amacwhirter) [reply src/components/TableHeadCell.js]: Sounds good - thanks!

**Comment by gabrielliwerant** [comment]
Looks good, thank you @amacwhirter! I will add this to the next release.

**Comment by KasparasGud** [comment]
Thank you  @amacwhirter  💪

**Pull request merged**